source.go

```diff
@@ -184,25 +187,35 @@ func source2LLBGetter(s *Spec, src Source, name string, forMount bool) LLBGetter
 			opts = append(opts, llb.Filename(name))
 			return llb.HTTP(https.URL, opts...), nil
 		case src.Context != nil:
-			st, err := sOpt.GetContext(dockerui.DefaultLocalNameContext, localIncludeExcludeMerge(&src))
+			name := src.Context.Name
+			if name == "" {
```

Comment:
I think this should be handled in `load.go` already?

Reply:
Done.